cli/test/smokehouse/lighthouse-runners/bundle.js

```diff
@@ -82,7 +82,7 @@ async function runBundledLighthouse(url, config, testRunnerOptions) {
     const logLevel = testRunnerOptions.isDebug ? 'verbose' : 'info';
 
     // Puppeteer is not included in the bundle, we must create the page here.
-    const browser = await puppeteer.connect({browserURL: `http://localhost:${port}`});
+    const browser = await puppeteer.connect({browserURL: `http://127.0.0.1:${port}`});
```

Comment:
Node no longer autoresolves localhost to the ipv4 address https://github.com/nodejs/node/issues/40702

I guess it does in Node 20 though so this is only an issue for Node 18